cdap-docs/faqs/source/general.rst

```diff
@@ -33,8 +33,9 @@ CDAP currently supports Java for developing applications.
 
 What version of Java SDK is required by CDAP?
 ---------------------------------------------
-The latest version of the JDK or JRE version 7 or version 8 must be installed
-in your environment; we recommend the Oracle JDK.
+The latest version of the JDK or JRE version 7 or version 8 must be installed in your
```

Comment:
It's only there because that was our recommendation when we only tested on Oracle. If we are now agnostic on the issue of which JDK to use, then it can go.